lib/hacluster.pm

```diff
@@ -101,6 +101,16 @@ Extension (HA or HAE) tests.
 
 =item * B<$crm_mon_cmd>: crm_mon (crm monitoring) command
 
+=item * B<$corosync_token>: command to filter the value of C<runtime.config.totem.token> from the output of C<corosync-cmapctl>
+
+=item * B<$corosync_consensus>: command to filter the value of C<runtime.config.totem.consensus> from the output of C<corosync-cmapctl>
+
+=item * B<$sbd_watchdog_timeout>: command to extract the value of C<SBD_WATCHDOG_TIMEOUT> from C</etc/sysconfig/sbd>
+
+=item * B<$sbd_delay_start>: command to extract the value of C<SBD_DELAY_START> from C</etc/sysconfig/sbd>
+
+=item * B<$pcmk_delay_max>: command to get the value of the C<pcmd_delay_max> parameter from the STONITH resource in the cluster configuration.
```

Comment:
```suggestion
=item * B<$pcmk_delay_max>: command to filter the value of the C<pcmd_delay_max> parameter from the STONITH resource in the cluster configuration.
```

Just top be coherent with other descriptions

Reply:
Yes, but in this particular case there's no filtering. Output of `crm resource param stonith-sbd show pcmk_delay_max` is only stripped of some characters with `sed 's/[^0-9]*//g'`. In the other cases the output of `corosync-cmapctl` is filtered by matching a specific text.